cypress/integration/userCanReadASpecificArticle.feature.js

```diff
@@ -58,4 +94,12 @@ describe("User can", () => {
       );
     });
   });
+
+  describe("NOT read article when they're not registered", () => {
```

Comment:
Make sure that this is restricted in your API as well. So not anything you need to change in this PR but take a note on updating your API